### PR TITLE
DF-1214 threshold defined

### PR DESCRIPTION
**Pull request by dvoracek-martin**

### CODE REVIEW

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/AvamIntegrationChannels.java [hunk 1]
```diff
@@ -33,7 +37,7 @@ public QueueChannel queueChannel() {
 
 			@Override
 			public int threshold() {
-				return 0;
+				return avamIntegrationThreshold;
```
Comment (birom) [inline infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/AvamIntegrationChannels.java]: Why do not we provide a default implementation in the alv-spring-integration-starter?

Comment (dvoracek-martin) [reply infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/AvamIntegrationChannels.java]: reimplemented and ready to be reviewed

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/AvamIntegrationChannels.java [hunk 2]
```diff
@@ -14,6 +15,9 @@
 
 	private final ChannelMessageStore channelMessageStore;
 
+	@Value("${avam.integration.threshold:20}")
+	private int avamIntegrationThreshold;
```
Comment (pavax) [inline infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/AvamIntegrationChannels.java]: not needed

File: build.gradle [hunk 3]
```diff
@@ -51,7 +51,10 @@ subprojects {
     repositories {
         maven { url 'http://repo.spring.io/milestone' }
         jcenter()
-        maven { url 'https://alvch.jfrog.io/alvch/libs-releases' }
+        maven { url 'https://alvch.jfrog.io/alvch/libs-releases-local' }
```
Comment (pavax) [inline build.gradle]: is this needed?

**Pull request merged**